_data/compatibility/opendime.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: Opendime
+internal_url: /en/compatibility/opendime
+logo: /img/compatibility/opendime/opendime.png
+rbf:
+  tested:
+    date: "2018-10-18"
+    platforms:
+      - macOS
+    version: "2.1.0"
+  features:
+    receive:
+      notification: "na"
+      list: "false"
+      details: "na"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "na"
+      shows_replaced_version: "na"
+      shows_original_version: "na"
+  examples:
+    - image: /img/compatibility/opendime/sending-instructions.png
+      caption: >
+        Sending RBF Transaction - Opendime does not have send transaction capabilities. You need to sweep the private key to use in a regular wallet.
+    - image: /img/compatibility/opendime/sending-script.png
+      caption: >
+        Sending RBF Transaction - There is a balance.py python script which can send a transaction using pycoin but does not specify RBF when creating a transaction.
+    - image: /img/compatibility/opendime/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Opendime uses a simple address explorer on opendime.com. No RBF Flag. OPENDIME’s simple transaction list explorer does not provide RBF flag notice. However there are links to other explorers on the page (Blockchain.info, Blocktrail, LocalBitcoins, BitInfoCharts, a BIP122 link)
+    - image: /img/compatibility/opendime/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Transaction list screen when a transaction has been bumped. Bottom transaction is the bumped fee transaction. Original transaction does not show up in list anymore.
+segwit:
+  tested:
+    date: "2019-07-24"
+    platforms:
+      - macOS
+    version: "2.2.0"
+  features:
+    receive:
+      p2sh_wrapped: "false"
+      bech32: "false"
+      default: "p2pkh"
+    send:
+      bech32: "na"
+      change_bech32: "na"
+      segwit_v1: "Sending transactions via Opendime software is not supported.
+      Private keys are used by user-selected wallet software to create transactions."
+      bech32_p2wsh: "na"
+  examples:
+    - image: /img/compatibility/opendime/segwit/receive-screen.png
+      caption: >
+        Opendime has a single P2PKH receive address. It is not capable of
```

Comment:
Can the balance.py program be used to send funds off the opendime?

Reply:
Yep, balance.py can spend. And can send to bech32 p2wsh and p2wpkh. Im going to change the comment here as well as the results, including "balance.py" as the wallet in this scenario.